CMSgov/onemac
# oy2-16332 - fix status actions per AC change

Story: https://qmacbis.atlassian.net/browse/OY2-16332
Endpoint: https://d1wm30li5xav2x.cloudfront.net/

### Details

Change to AC to allow Submitted status Temp extensions to be withdrawn.

### Changes

- Altered status array in changeRequest.js


### Test Plan

1. Login as state submitter
2. Submit a new Temporary Extension on a Waiver.
3. Verify the temporary extension can be withdrawn from submission list and package dashboard
